src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -639,7 +639,7 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
         const cellMatcher = new CellMatcher(settings);
         const isMarkdown = cellMatcher.getCellType(code) === MARKDOWN_LANGUAGE;
         const strippedCode = isMarkdown
-            ? generateMarkdownFromCodeLines(code.splitLines()).join('\n')
+            ? generateMarkdownFromCodeLines(code.splitLines()).join('')
```

Comment:
generateMarkdownFromCodeLines calls appendLineFeed at the end on all the lines, so the join added on a second linefeed. The extra /n/n causes a number of markdowns to not work correctly.